server/api_server/schemas.py

```
@@ -0,0 +1,625 @@
+        ...,
+        description="Base64-encoded SQL query. (UNSAFE: Use only for administrative tasks)"
+    )
+    confirm_dangerous_query: bool = Field(
+        False,
+        description="Required to be True to acknowledge the risks of raw SQL execution"
+    )
+
+    @model_validator(mode="after")
+    def validate_safety(self) -> DbQueryRequest:
+        if not self.confirm_dangerous_query:
+            raise ValueError("You must set confirm_dangerous_query=True to execute raw SQL")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)